core/data/match.go

```diff
@@ -68,25 +68,26 @@ func getLogFileNameFor(displayName string, id string) string {
 func NewTrackingState(cfn string) *TrackingState {
 	return &TrackingState{
 		CFN:               cfn,
```

Comment:
Should change these to be "displayName" and "code" to be consistent with rest of the code base